src/toolbars/MeterToolBar.cpp

```diff
@@ -40,6 +40,87 @@
 #include "../ProjectAudioIO.h"
 #include "../widgets/MeterPanel.h"
 
+#if wxUSE_ACCESSIBILITY
+
+class MeterButtonAx : public AButtonAx
+{
+   wxWeakRef<MeterPanel> mAssociatedMeterPanel{nullptr};
+   bool mAccSilent{false};
+public:
+
+   MeterButtonAx(AButton& button, MeterPanel& panel)
+      : AButtonAx(&button)
+      , mAssociatedMeterPanel(&panel)
+   { }
+
+   wxAccStatus GetName(int childId, wxString* name) override
+   {
+      if(childId != wxACC_SELF)
+         return wxACC_NOT_IMPLEMENTED;
+
+      if(mAccSilent)
+         *name = wxEmptyString; // Jaws reads nothing, and nvda reads "unknown"
+      else
+      {
+         const auto button = static_cast<AButton*>(GetWindow());
+
+         *name = button->GetName();
+
+         if(const auto panel = mAssociatedMeterPanel.get())
+         {
+            // translations of strings such as " Monitoring " did not
+            // always retain the leading space. Therefore a space has
+            // been added to ensure at least one space, and stop
+            // words from being merged
+            if(panel->IsMonitoring())
+               *name += wxT(" ") + _(" Monitoring ");
```

Comment:
If you add the space yourself, for the special reasons explained in the comment, then do not include the spaces in the strings to be translated.

Reply:
This applies to five translatable strings in this function.